valohai_yaml/utils/upload_store.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from valohai_yaml.excs import ValidationError
+
+
+def parse_upload_store(upload_store: Optional[str]) -> Optional[str]:
+    if not upload_store:
+        return None
+    import re
+
+    if (isinstance(upload_store, str) and len(upload_store) <= 64) or re.match(
```

Comment:
The re.match part of this expression will never be visited for strings that could be valid uuids, since they'll be less than 64 characters long.